tests/utils/validation.spec.ts

```diff
@@ -186,26 +190,26 @@ describe('validateUuid() function', () => {
 
 describe('parseAsInteger() function', () => {
 	it('should reject non-numeric characters', () => {
-		expect(() => v.parseAsInteger('abc')).to.throw(ExpectedError);
```

Comment:
Previously we had a promise returning function also synchronously throwing errors, so I had to update this to handle proper async errors